Dockerfile

```
@@ -5,7 +5,7 @@ FROM python:${PYTHON_VERSION}
 ENV PIP_OPTIONS "--no-cache-dir --progress-bar off"
 
 RUN apt-get update \
-    && apt-get -y install openmpi-bin libopenmpi-dev \
+    && apt-get -y install openmpi-bin libopenmpi-dev libopenblas-dev\
```

How about adding a whitespace before `\` like the other lines?
```suggestion
    && apt-get -y install openmpi-bin libopenmpi-dev libopenblas-dev \
```